spec/requests/api/braintree_spec.rb

```diff
@@ -4,15 +4,22 @@
 VCR.configure do |config|
   config.cassette_library_dir = "spec/fixtures/vcr_cassettes"
   config.hook_into :webmock
+
+  # The filter_sensitive_data configuration option prevents
+  # sensitive data from being written to your cassette files.
+  #
+  %w{BRAINTREE_MERCHANT_ID BRAINTREE_PUBLIC_KEY BRAINTREE_PRIVATE_KEY}.each do |env|
+    config.filter_sensitive_data("<#{env}>") { ENV[env] }
+  end
```

Comment:
I had forgotten to filter out the sandbox keys from the cassette. The mocked request to braintree was expecting certain values for merchant ID, etc... this is why this spec was failing for you. Above, I've filtered the three keys and updated the cassette file. It should now pass for you.